content/blog/analyzing-the-success-of-the-hpe-dev-workshops-on-demand.md

```diff
@@ -0,0 +1,101 @@
+
+**Reason #1**
+
+They allow us to see if our content is relevant to the participant. In the completed surveys, participants gave the workshops an average rating of
+
+* **4.64 stars out of 5 for technical level**
+* **4.63 stars out of 5 for the overall value of the workshop**
+
+**Reason #2**
+
+They allow us to see if the delivery method is correct. The surveys indicated that the Jupyter Notebooks format has been very well accepted, with an average rating of
+
+* **4.60 stars out of 5**
+
+**Reason #3**
+
+They help us define new topics and give us insight into the user’s experience. Comments such as “*Great initiative, the more technical format is the way to go for us engineers and developers, just try to go deeper (using more time if required)*.”and “*Great idea and a very suitable form of delivery.*” provide feedback that we can use to modify content and add additional, appropriate courses.
+
+ 
+
+**Content**
+
+Aligning with the HPE DEV charter, the workshops span topics that interest our audience, which tend to be related to:
+
+* IT OPS and DevOPs in terms of API interaction, Automation, or Programming
+* Open source
+* Coding languages
+* Data analytics
+* AI /MLOPS
+
+The workshops range from simple 101-level sessions to more advanced labs. We try to keep a balance of courses available for participants who are just starting to courses for those community members who are looking for more technical depth.
+
+As mentioned earlier, we started small and have continued to build out a wide selection of workshops over this past year. We plan to continue to expand this program, always taking your feedback on which new topics should be offered.
+
+![](/img/wodanalysisblog4.png)
+
+Some topics tend to be more popular than others. You can see that the HPE Ezmeral workshops are very well attended. AI /MLOPS and HPE iLO/Redfish workshops are also quite popular. Open Source and Programming workshops follow close behind.
+
+ 
+
+**Infrastructure**
+
+We started the program with a single JupyterHub server to support the very first workshops. Each of the workshops are using different types of API endpoints. When reaching a public weather forecasting API, nothing much is required on the backend side apart from the JupyterHub server itself.
+
+When targeting an HPE Ezmeral Container Platform API, however, the requirements are completely different. To face an ever growing number of workshops and participants, we decided to implement additional Jupyter servers. In addition to the first JupyterHub server sitting in an HPE site in France, we created a second Jupyter server in another HPE location. We also leverage HPE GreenLake cloud services to offload some workshops. We now have three production sites, which allow us improve our workshop’s resiliency. We can now easily redirect an existing workshop from one site to another.
```

Comment:
be consistent, use JupyterHub servers everywhere. Allow us to improve (missing to)